software/mediawiki.yml

```
@@ -1,6 +1,6 @@
 name: Mediawiki
 website_url: https://www.mediawiki.org/wiki/MediaWiki
-description: MediaWiki is a free and open-source wiki software package written in PHP. It serves as the platform for Wikipedia and the other Wikimedia projects, used by hundreds of millions of people each month.
+description: Wiki software package serving as the platform for Wikipedia and the other Wikimedia projects, used by hundreds of millions of people each month.
```

More sensible description
```suggestion
description: Wiki software package that powers Wikipedia and all other Wikimedia projects, serving hundreds of millions of users each month.
```